frontend/packages/core/src/Assests/emojis/happy-emoji.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+
+import type { SVGProps } from "../global";
+import { StyledSVG } from "../global";
+
+const HappyIcon = ({ size }: SVGProps) => (
+  <StyledSVG size={size} viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
can we pass through the remaining props?
```suggestion
const HappyIcon = ({ size, ...props }: SVGProps) => (
  <StyledSVG size={size} viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
```